server/channels/api4/system.go

```diff
@@ -120,7 +121,7 @@ func generateSupportPacket(c *Context, w http.ResponseWriter, r *http.Request) {
 	// We do this incase we get concurrent requests, we will always have a unique directory.
 	// This is to avoid the situation where we try to write to the same directory while we are trying to delete it (further down)
 	outputDirectoryToUse := OutputDirectory + "_" + model.NewId()
-	err := c.App.CreateZipFileAndAddFiles(fileStorageBackend, fileDatas, outputZipFilename, outputDirectoryToUse)
+	err := c.App.CreateZipFileAndAddFiles(&request.Context{}, fileStorageBackend, fileDatas, outputZipFilename, outputDirectoryToUse)
```

Comment:
Please just reuse the existing context
```suggestion
	err := c.App.CreateZipFileAndAddFiles(c.AppContext, fileStorageBackend, fileDatas, outputZipFilename, outputDirectoryToUse)
```